test/unit/Reflection/ReflectionClassConstantTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Roave\BetterReflectionTest\Reflection;
+
+use Roave\BetterReflection\Reflector\ClassReflector;
+use Roave\BetterReflection\SourceLocator\Type\ComposerSourceLocator;
+use Roave\BetterReflectionTest\Fixture\ExampleClass;
+
+class ReflectionClassConstantTest extends \PHPUnit_Framework_TestCase
+{
+    private function getComposerLocator() : ComposerSourceLocator
+    {
+        global $loader;
```

Comment:
Yep, is really smelly code, but...
https://github.com/Roave/BetterReflection/blob/master/test/unit/Reflection/ReflectionClassTest.php#L44 :(